pkg/orchestrator/etcd_worker.go

```diff
@@ -120,9 +120,20 @@ func (worker *EtcdWorker) initMetrics() {
 // And the specified etcd session is nil-safety.
 func (worker *EtcdWorker) Run(ctx context.Context, session *concurrency.Session, timerInterval time.Duration, role string) error {
 	defer worker.cleanUp()
+
+	if role == pkgutil.RoleOwner.String() {
```

Comment:
maybe we can put these 3 line into checkAndMigrateMetaData func

Reply:
ok